exercises/chapter10/src/Data/AddressBook.purs

```diff
@@ -26,11 +26,12 @@ data PhoneType
   | OtherPhone
 
 -- ANCHOR: PhoneType_generic
-derive instance genericPhoneType    :: Generic    PhoneType _
-instance        encodeJsonPhoneType :: EncodeJson PhoneType where encodeJson = genericEncodeJson
-instance        decodeJsonPhoneType :: DecodeJson PhoneType where decodeJson = genericDecodeJson
+derive instance genericPhoneType :: Generic PhoneType _
```

Comment:
I think it's clearer to keep alignment of the derived instance separate from the other instance definitions.